pkg/operator/osd/osd.go

```diff
@@ -214,28 +219,37 @@ func (c *Cluster) osdContainer(devices []Device, directories []Directory, select
 		k8sutil.ConfigOverrideEnvVar(),
 	}
 
+	devMountNeeded := false
```

Comment:
i was actually referring to set this to true in the the `if` condition in line 168 but i guess this would work too.

Reply:
line 168 is another function, and it only needs that decision once so I did not assign a variable.